template/web/src/lib/Code.svelte

```diff
@@ -0,0 +1,75 @@
+<script>
+  import { onMount } from 'svelte'
+  export let portableText
+
+  $: block = portableText.block
+
+  onMount(() => {
+
+   let script = document.createElement('script');
+   script.src = "https://tutsplus.github.io/syntax-highlighter-demos/highlighters/Prism/prism.js"
+   document.head.append(script);
```

Comment:
What are your thoughts on [svelte-prism](https://github.com/jakobrosenberg/svelte-prism)? I was hoping it could simplify the component, but aren't sure of your motivation :) 

If we do choose to go with our custom implementation, we should consider putting the script loading logic in a global store to prevent loading Prism more than once 🤔

Reply:
I've never used that component but it looks great! I just pulled this method off of a REPL, but using this comp would be a lot cleaner.